### PR TITLE
Feat: use liste detail on feature click

### PR DESCRIPTION
**Pull request by mbarbet**
- Fix https://github.com/gisaia/ARLAS-wui/issues/673

### CODE REVIEW

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 1]
```diff
@@ -1124,50 +1129,75 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
...
-            const popupContent = this.dynamicComponentService.injectComponent(
-              ResultDetailedItemComponent,
-              x => {
-                x.rowItem = rowItem;
-                x.actionOnItemEvent = this.actionOnPopup;
-                x.idFieldName = idFieldName;
-              });
-            if (!!this.popup) {
-              this.popup.remove();
+        // Open the list panel if it's close
```
Comment (QuCMGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1132]: close -> closed

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 2]
```diff
@@ -1124,50 +1129,75 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
...
+        this.disableRecalculateExtend = true;
+        if (!this.resultlistService.listOpen) {
+          this.toggleList();
+        }
+        // Selec the good tab if we have several one
```
Comment (QuCMGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1137]: selec -> select
several one -> several

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 3]
```diff
@@ -1124,50 +1129,75 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
...
+        // No tabs case
+        if (this.resultlistContributors.length === 1) {
+          setTimeout(() => {
```
Comment (QuCMGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1140]: why do you need a timeout ?

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 4]
```diff
@@ -1124,50 +1129,75 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
...
+      }
+    }
+  }
+
+  private openDetail(id: any) {
+    const isListMode = this.resultListComponent.resultMode === this.modeEnum.list;
+    if (isListMode) {
+      const detailListButton = document.getElementById('open-detail-' + id);
+      if (!!detailListButton) {
+        // close previous if exist
```
Comment (QuCMGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1161]: exist -> exists

**Review by QuCMGisaia — APPROVED** [review]

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts (hunk 3 above)
Comment (QuCMGisaia) [reply src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1140]: Is there a way to wait for a specific event instead of using a timeout ?

Comment (mbarbet) [reply src/app/components/arlas-wui-root/arlas-wui-root.component.ts]: @QuCMGisaia is it ok for you ?

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts [hunk 5]
```diff
@@ -1124,50 +1129,74 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
...
+        if (this.resultlistContributors.length === 1) {
+          this.waitFor(this.resultListComponent,() => this.openDetail(id));
+        } else {
+          this.waitFor(this.resultListComponent,() => {
+            // retrieve list
+            const tab = document.querySelector('[aria-label="' + resultListContributor.identifier + '"]') as any;
+            tab.click();
+            // Set Timeout to wait the new tab
+            setTimeout(() => this.openDetail(id), 250);
+            this.disableRecalculateExtend = false;
```
Comment (MohamedHamouGisaia) [inline src/app/components/arlas-wui-root/arlas-wui-root.component.ts:1148]: this disabling could happen early as the `openDetail` is executed after and `openDetail` it self has timeouts inside, no ?

File: src/app/components/arlas-wui-root/arlas-wui-root.component.ts (hunk 3 above)
Comment (QuCMGisaia) [reply src/app/components/arlas-wui-root/arlas-wui-root.component.ts]: Yes that is perfect !

**Review by MohamedHamouGisaia — APPROVED** [review]

**Pull request merged**